databricks_cli/pipelines/cli.py

```diff
@@ -286,10 +421,10 @@ def _gen_start_update_msg(resp, pipeline_id, full_refresh):
     return output_msg
 
 
-def _read_spec(src):
+def _read_settings(src):
     """
-    Reads the spec at src as a JSON if no file extension is provided, or if in the extension format
-    if the format is supported.
+    Reads the settings at src as a JSON if no file extension is provided,
+    or if in the extension format if the format is supported.
```

Comment:
it does not like this is accurate. It requires `.json`

Reply:
updated the code to also handle no extension case to be considered as json file.